### PR TITLE
📝 Icon documentation

### LINKED ISSUE #446: Create documentation for Icon
Write documentation for [Icon](https://eds.equinor.com/components/icon/)

**Sub tasks**

- [ ] Write documentation
- [ ] Create examples
- [ ] Add props-table
- [ ] Add live preview

### PR DESCRIPTION
**Pull request by martalalik**
Resolves #446

### CODE REVIEW

**Comment by martalalik** [comment]
I'm not sure to the `Size` description. 
In the example we have icon sizes from `16px` to `48px`, while in [documentation](https://eds.equinor.com/0b0c666ab/p/347dd6-system-icons/b/25520a) there are mention sizers form `16px` to `24px`. 🤔

File: packages/eds-core-react/src/components/Icon/Icon.docs.mdx [hunk 1]
```diff
@@ -59,6 +67,12 @@ If you are using the icons alone to convey meaning you need to provide the `titl
 
 
 ## Sizes
+Simple and clear icons require a strict underlying grid.
+<ul>
+    <li>The EDS icons are built on a 24x24px grid.</li>
+    <li>System icons can be used in 24px and 16px sizes.</li>
```
Comment (mimarz) [inline packages/eds-core-react/src/components/Icon/Icon.docs.mdx:73]: Suggest removing these two paragraphs as the sizes in props table speaks for them selves. Good to mention when its recommended to use 18 px as its the odd one out

**Review by mimarz — CHANGES_REQUESTED** [review]

File: packages/eds-core-react/src/components/Icon/Icon.stories.tsx [hunk 2]
```diff
@@ -27,7 +27,12 @@ export default {
   },
 } as ComponentMeta<typeof Icon>
 
-export const Introduction: Story<IconProps> = (args) => <Icon {...args} />
+export const Introduction: Story<IconProps> = (args) => (
+  <Stack>
+    {' '}
```
Comment (mimarz) [inline packages/eds-core-react/src/components/Icon/Icon.stories.tsx:32]: Remove space blunder

Comment (mimarz) [inline packages/eds-core-react/src/components/Icon/Icon.stories.tsx:32]:
```suggestion
```

**Pull request merged**